pipelines/run-pipelines-on-apache-airflow/README.md

```diff
@@ -0,0 +1,332 @@
+    - Host: `https://api.github.com`
+    - Schema: `https` 
+    
+![](doc/images/airflow-connection-id.png)
+    
+#### Clone the tutorial artifacts
+This tutorial uses the `run-pipelines-on-apache-airflow` sample from the https://github.com/elyra-ai/examples GitHub repository.
+
+1. Launch JupyterLab.
+1. Open the _Git clone_ wizard (Git > Clone A Repository).
+1. Enter `https://github.com/elyra-ai/examples.git` as _Clone URI_.
+1. In the _File Browser_ navigate to `examples/pipelines/run-pipelines-on-apache-airflow`.
+
+   ![Tutorial assets in File Browser](doc/images/tutorial-files.png)
+   
+   The cloned repository includes the resources needed to run the tutorial pipeline.
+
+You are ready to start the tutorial.
+
+### Add custom components via component catalog
+
+Elyra stores information about custom components in component catalogs and makes those components available in the Visual Pipeline Editor's palette. Components can be grouped into categories to make them more easily discoverable.
+
+![Pipeline editor palette with components](doc/images/palette-in-pipeline-editor.png)
+
+Custom components are [managed in the JupyterLab UI](https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html#managing-pipeline-components) using the **Pipeline components** panel. You access the panel by:
+   - Selecting `Component Catalogs` from the JupyterLab sidebar.
+   - Clicking the `Open Component Catalogs` button in the pipeline editor toolbar.
+   - Searching for `Manage URL Component Catalog`, `Manage Filesystem Component Catalog`, or `Manage Directory Component Catalog` in the [JupyterLab command palette](https://jupyterlab.readthedocs.io/en/stable/user/commands.html).
+
+> You can automate the component management tasks using the [`elyra-metadata install component-catalog` CLI command](https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html#managing-custom-components-using-the-elyra-cli). 
+
+The component catalog can access component specifications that are stored in the local file system or on remote sources. In this tutorial 'local' refers to the file system where JupyterLab/Elyra is running. For example, if you've installed Elyra on your laptop, local refers to the laptop's file system. If you've installed Elyra in a container image, local refers to the container's file system.
+
+#### Add components from local sources
+
+To add component specifications to the registry that are stored locally:
+
+1. Open the **Component Catalogs** panel using one of the approaches mentioned above.
+
+   ![Add component registry entry](doc/images/add-component-catalog-entry.png)
+
+1. Add a new component registry entry by clicking `+` and selecting `New Filesystem Component Catalog`.
+   The first tutorial component you are adding to the registry makes an HTTP Request.
+1. Enter or select the following:
+   - **Name**: `request data`
+   - **Description**: `request data from GitHub API`
+   - **Category**: `request`
```

Comment:
'Runtime' is displayed before 'category' in the GUI. (Need to switch the order here)